Doc/Manual/Python.html

```diff
@@ -4134,8 +4134,9 @@ <H3><a name="Python_stable_abi">33.6.6 Stable ABI</a></H3>
 
 <p>
 The default SWIG command line options generate code that enable the limited API/stable ABI.
-Some options, such as <tt>-builtin</tt>, <tt>-fast</tt> (used by <tt>-O</tt>) do not use the limited API and hence when <tt>Py_LIMITED_API</tt> is defined there will be missing symbols during compilation.
+Some options, such as <tt>-builtin</tt>, <tt>-fastproxy</tt> (used by <tt>-O</tt>) do not use the limited API and hence when <tt>Py_LIMITED_API</tt> is defined there will be missing symbols during compilation.
 Compatibility of user's custom typemaps is of course dependent on the Python APIs used in the typemaps.
+If you disable <tt>-O</tt>, you could consider enabling <tt>-fastdispatch</tt> as this option is abi3 compatible.
```

Comment:
@wsfulton removed the part about needing `-fastdispatch`, but left in some mention of it. Do you think this is worthwhile?